crates/fluvio/Cargo.toml

```diff
@@ -15,14 +15,17 @@ name = "fluvio"
 path = "src/lib.rs"
 
 [features]
-default = ["rustls", "compress"]
+default = ["rustls", "compress", "aws-lc-rs"]
 admin = ["fluvio-sc-schema/use_serde"]
 smartengine = ["fluvio-smartengine"]
 openssl = ["fluvio-future/openssl_tls"]
-rustls = ["fluvio-future/rust_tls"]
+rustls = ["fluvio-future/rust_tls", "dep:rustls"]
 compress = ["fluvio-compression/compress", "fluvio-protocol/compress"]
 nightly = []
 unstable = []
+# Crypto providers for rustls (mutually exclusive)
+aws-lc-rs = ["rustls?/aws-lc-rs", "rustls?/prefer-post-quantum"]
```

Comment:
can we get use consistent feature name?  it should be something like `rustls-aws`,  `rusts-ring`.

Reply:
Sure, I'll create a new PR for that.